test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -141,8 +141,10 @@ List<IReportMetadata> InitializeReportMetadataList()
                 new CountingReportMetadata("relayer1.send", "relayer1.eventHub", TestOperationResultType.Messages, TestReportType.CountingReport),
                 new CountingReportMetadata("loadGen2.send", "relayer2.receive", TestOperationResultType.Messages, TestReportType.CountingReport),
                 new CountingReportMetadata("relayer2.send", "relayer2.eventHub", TestOperationResultType.Messages, TestReportType.CountingReport),
-                new CountingReportMetadata("directMethodSender1.send", "directMethodReceiver1.receive", TestOperationResultType.DirectMethod, TestReportType.CountingReport),
-                new CountingReportMetadata("directMethodSender2.send", "directMethodReceiver2.receive", TestOperationResultType.DirectMethod, TestReportType.CountingReport),
+                // TODO: Enable Direct Method Cloud-to-Module and Cloud-to-EdgeAgent once the verification scheme is finalized.
```

Comment:
Can you clarify why we need to comment this out?

Reply:
The verification & report generating part is not there yet. 
It has a dependency on this PR: https://github.com/Azure/iotedge/pull/2300/files

Reply:
It is better to comment it out since these counting report should not be used for DM scenario which created noise in test report result.